zerver/views/auth.py

```diff
@@ -34,6 +40,55 @@
 import requests
 import time
 import ujson
+import datetime
+from typing import Text
+
+def send_find_my_team_emails(user_profile):
+    # type: (UserProfile) -> None
+    path = 'zerver/emails/find_team/'
+    text_template = path + 'find_team_email.txt'
+    html_template = path + 'find_team_email_html.txt'
+    context = {'user_profile': user_profile}
+    text_content = loader.render_to_string(text_template, context)
+    html_content = loader.render_to_string(html_template, context)
+    sender = {'email': settings.NOREPLY_EMAIL_ADDRESS, 'name': 'Zulip'}
+    recipients = [{'email': user_profile.email, 'name': user_profile.full_name}]
+    subject = "Your Zulip Team"
+
+    # Send now, through Mandrill.
+    send_future_email(recipients, html_content, text_content, subject,
+                      delay=datetime.timedelta(0), sender=sender,
+                      tags=["find-my-team-emails"])
```

Comment:
Why are you using `send_future_email` here?  I think maybe you just want `send_mail`